src/main/java/com/disraely/entities/BaseEntity.java

```diff
@@ -0,0 +1,35 @@
+package com.disraely.entities;
+
+import java.io.Serializable;
+
+public class BaseEntity implements Serializable {
```

Comment:
Is this this base entity a type of entity? What are the benefits of instancing the `Base entity`?

Reply:
The idea is that BaseEntity is a parent of all entities, to centralize common attributes/methods.

Benefits:

1. This class seeks to centralize all the common attributes of all its children.
2. Access for the massive modification in the system for all entities. 
3. An entity standard is maintained (all entities must have an ID, createDate, ...).

Reply:
I converted BaseEntity into an abstract class so that it cannot be instantiated directly and has to be inherited by another one.